LittleMouseGames/GodotAssetLibrary
# Seo 26 update general seo

[Project Link](https://github.com/LittleMouseGames/GodotAssetLibrary/issues/26)

### Goal:
Update parts of the site for better SEO

### Testing Instructions
- Make sure the site works as expected
- Make sure badges on cards link to `/category/` and `/engine/` URLs
  - verify the filtering, searching, pagination on here works as we expect it to

### Resources
N/A

### Checklist
- [x] The changes are scoped appropriately
- [x] The project builds and runs successfully
- [x] I have self reviewed the changes before assigning reviewer
